sdk/spring/azure-spring-data-cosmos/src/test/java/com/azure/spring/data/cosmos/domain/Address.java

```diff
@@ -9,7 +9,7 @@
 
 import java.util.Objects;
 
-@Container()
+@Container(ru = TestConstants.DEFAULT_MINIMUM_RU)
```

Comment:
why we will have to specifically configure the RU here (default is 400 I think )